docs/4_contributing/developer-guide.md

```diff
@@ -1,70 +1,68 @@
----
-title: Developer Guide
-toc: true
-weight: 710
----
 # Developer Guide
```

Comment:
This doc feels a bit outdated and I think the new concepts may be covered elsewhere throughout the new docs, @prasek do you feel okay about this being removed?

Reply:
@prasek following up here, okay to remove?

Reply:
yes, good to remove, thx